Tests/FoundationTests/URLExtensionsTests.swift

```diff
@@ -71,4 +71,17 @@ final class URLExtensionsTests: XCTestCase {
 	}
 	#endif
 
+    func testDropScheme() {
+        let urls: [String: String] = [
+            "https://domain.com/path/other/": "domain.com/path/other/",
+            "https://domain.com": "domain.com",
+            "http://domain.com": "domain.com",
+            "file://domain.com/image.jpeg": "domain.com/image.jpeg"
```

Comment:
I suggest adding some of the urls listed in the original issue. E.g.
```
"://apple.com"
"//apple.com"
"apple.com"
```

Reply:
Also make sure to test `"http://"`.
And what do you think should be the outcome of `//`? Should it be `nil` or `//`? Currently it returns the latter, but to me it makes more sense for the former. Not an important edge case, but it's a curious one.

Reply:
@guykogus 

I've added test for `http://`.

I believe outcome of `//` should be `nil`, however I would need to add specific condition to detect this case, otherwise I'm unable to detect it. Shall I do that?

Reply:
No, I don't think it's necessary.